client/src/Login.js

```diff
@@ -2,40 +2,44 @@ import { Stytch } from "@stytch/stytch-react";
 import React from "react";
 
 const Login = () => {
-  const STYTCH_PUBLIC_TOKEN = "public-token-1111-1111-1111";
+  const STYTCH_PUBLIC_TOKEN =
+    "public-token-test-196bf55b-4d4d-4694-bb9c-4312a2398cf6";
```

Comment:
Please revert this change

Reply:
sorry, forgot to revert changes to this file

fixed: https://github.com/stytchauth/stytchjs-react-express/pull/14/commits/0f346ca34b958a86d4b8c1abcade734d0a37d7d1